.gitignore

```diff
@@ -90,3 +90,6 @@ src/GovUk.Education.ExploreEducationStatistics.Admin/appsettings.IdpBootstrapUse
 
 ## IdentityServer Developer Signing key
 **/*/tempkey.rsa
+
+## Local Gov.UK Notify integration settings
+/src/GovUk.Education.ExploreEducationStatistics.Notifier/local.settings.json
```

Comment:
In the comment I would call this `Notifier Local settings file` because it contains more than just the Gov.uk Notify integration settings.

You could also put this in a new `.gitignore` file in the root of the Notifier project. We have mutliple `.gitignore` files so there's already precedent for doing this.